src/Identifier/IdentifierType.php

```diff
@@ -46,11 +46,21 @@ public function getName()
      */
     public function getDisplayName()
     {
-        return ucfirst(basename($this->name));
+        return basename(str_replace('\\', '/', $this->name));
```

Comment:
yes, exactly, it turns `\BetterReflection\Reflection\ReflectionClass` into just `ReflectionClass` for example